canso-data-plane/canso-aws-eks-superchart/values.yaml

```diff
@@ -8,6 +8,16 @@ argocdProject: canso-appsets # WARNING: CRITICAL VALUE - DO NOT CHANGE
 ## @param createProject Create the ArgoCD project.
 createProject: true
 
+## @section ArgoCD AI Agent Configs
+##
+
+## @param argocdAIAgentProject ArgoCD AI Agent project name.
+##
+argocdAIAgentProject: canso-apps # WARNING: CRITICAL VALUE - DO NOT CHANGE
+
+## @param createAIAgentProject Create the ArgoCD AI Agent project.
+createAIAgentProject: true
```

Comment:
This should be mentioned before `argocdAIAgentProject`. We'll need the project name only if create project is set to `True`.